aries-site/src/data/structures/templates.js

```diff
@@ -598,3 +598,128 @@ export const templates = [
     tags: ['content', 'layout', 'page layouts', 'responsiveness', 'responsive'],
   },
 ];
+
+// these pages do not necessarily have their own routes but instead
+// link to sections of existing pages
+export const contentLayouts = [
```

Comment:
This struck me as an odd fit to live in this file. I like the idea of keeping this file focussed solely on templates pages. While this content is related to a templates page, I think it deviates from the rest of the established pattern. I'd rather see it contained in its own dedicated file.

We have most of our page assets stored in the `/examples` directory. If I were wanting to make updates to the Content Layouts page, I would probably look there or `/pages` first. I'm wondering if this should live within the `/examples/content-layouts` directory? Perhaps under `/examples/content-layouts/data/`?

Reply:
That's fair -- since these were page objects I was debating if it should live closer to structures as opposed to data that is used within examples. what would you think of `/data/structures/content-layouts.js`?

Reply:
Gotcha. I guess I was viewing them not as page objects (though their properties are highly similar) since they don't represent pages, rather they represent content to which we are cross-linking, similar to https://design-system.hpe.design/components/card#dashboard-example.

I'm okay going with either:
- `/data/structures/templates/content-layouts.js`
- `/data/structures/templates/content-layouts/relatedContent.js` 
- `/data/structures/templates/content-layouts/relatedExamples.js`
- `/examples/templates/content-layouts/data/relatedContent.js` or similar

Side note, just found [this](https://design-system.hpe.design/components/card#navigation-example) --> Click on the link to go to the navigation example.